### PR TITLE
Add docs for Volumes UI updates

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Draft docs for Volumes UI updates

### CODE REVIEW

File: desktop/dashboard.md [hunk 1]
```diff
@@ -227,23 +227,40 @@ To interact with remote repositories:
 
 ## Explore volumes
 
-The **Volumes** view in Docker Dashboard enables you to easily create and delete [volumes](../storage/volumes.md) and see which ones are being used. If you are a developer subscribed to a Pro or a Team plan, you can also see which container is using a specific volume and explore the files and folders in your volumes.
+A [volume](../storage/volumes.md) is a directory on a disk or in a container. Volumes are directly managed by Docker and are stored as part of the host file system.
```
Comment (mat007) [inline desktop/dashboard.md:230]: It’s actually the other way around: volumes are directories stored in the VM (also but rarely called the guest file system, mostly internally at Docker I think) while bind mounts are directories (or files) mounted from the host.

File: desktop/dashboard.md [hunk 2]
```diff
@@ -227,23 +227,40 @@ To interact with remote repositories:
...
 
+The following table lists the Volumes UI features available in each plan:
+
+{% assign yes = '![yes](/images/green-check.svg){: .inline style="height: 14px; margin: 0 auto"}' %}
+
+| Feature      | Free | Pro | Team   |
+|:-------------|:---------:|:----------:|:----------:|
+|  View, create, and delete volumes| {{ yes }}    | {{ yes }}   |  {{ yes }}    |
+| See which volumes are in use | {{ yes }} | {{ yes }}   |  {{ yes }}    |
+| See the size of the volume | {{ yes }}  | {{ yes }}   |  {{ yes }}    |
```
Comment (mat007) [inline desktop/dashboard.md:248]: nit: would be easier to review if the `|` were aligned 😁

File: desktop/dashboard.md [hunk 3]
```diff
@@ -227,23 +227,40 @@ To interact with remote repositories:
...
+| See volume data details (modified date, size, kind) | X | {{ yes }}   |  {{ yes }}    |
+| See which container is using a volume | X | {{ yes }}   |  {{ yes }}    |
+| Save or delete data from a volume | X| {{ yes }}   |  {{ yes }}    |
```
Comment (mat007) [inline desktop/dashboard.md:251]: nit: maybe `data` -> `files`?

File: desktop/dashboard.md [hunk 4]
```diff
@@ -227,23 +227,40 @@ To interact with remote repositories:
...
+The following table lists the Volumes UI features available in each plan:
+
+{% assign yes = '![yes](/images/green-check.svg){: .inline style="height: 14px; margin: 0 auto"}' %}
+
+| Feature      | Free | Pro | Team   |
+|:-------------|:---------:|:----------:|:----------:|
+|  View, create, and delete volumes| {{ yes }}    | {{ yes }}   |  {{ yes }}    |
+| See which volumes are in use | {{ yes }} | {{ yes }}   |  {{ yes }}    |
+| See the size of the volume | {{ yes }}  | {{ yes }}   |  {{ yes }}    |
+| See volume data details (modified date, size, kind) | X | {{ yes }}   |  {{ yes }}    |
```
Comment (stephanierifai) [inline desktop/dashboard.md:249]: Sorry, I forgot we also removed the kind column!

```suggestion
| See volume data details (modified date, size) | X | {{ yes }}   |  {{ yes }}    |
```

**Review by stephanierifai — COMMENTED** [review]
Looks good - just one suggestion because I remembered we removed the kind column.

File: desktop/dashboard.md [hunk 5]
```diff
@@ -227,23 +227,40 @@ To interact with remote repositories:
...
+By default, the **Volumes** view displays a list of all the volumes. Volumes that are currently used by a container display the **In Use** badge.
 
 ![List volumes](images/volumes-list.png){:width="700px"}
 
+Use the **Search** field to search for any specific volumes. You can also sort volumes by the name, the date created, and the size of the volume.
+
 To explore the details of a specific volume, select a volume from the list. This opens the detailed view.
 
-The **In Use** tab displays the name of the container that’s using the volume, the image name, the port number used by the container, CPU, memory, disk read/write, and network I/O consumed by the container.
+The **In Use** tab displays the name of the container using the volume, the image name, the port number used by the container, and the target. A target is a mountpoint inside a container that gives access to the file system on the host.
```
Comment (mat007) [inline desktop/dashboard.md:263]:
```suggestion
The **In Use** tab displays the name of the container using the volume, the image name, the port number used by the container, and the target. A target is a path inside a container that gives access to the volume files.
```
(it’s not on the host, it’s inside the VM)

**Review by mat007 — CHANGES_REQUESTED** [review]

File: desktop/dashboard.md (hunk 4 above)
Comment (usha-mandya) [reply desktop/dashboard.md]: Thanks Steph. Updated.

File: desktop/dashboard.md (hunk 2 above)
Comment (usha-mandya) [reply desktop/dashboard.md]: Fixed... ( Almost 😀 )

File: desktop/dashboard.md (hunk 1 above)
Comment (usha-mandya) [reply desktop/dashboard.md]: Updated the description. PTAL and let me know if we should further amend this.

File: desktop/dashboard.md (hunk 3 above)
Comment (usha-mandya) [reply desktop/dashboard.md]: Fixed

File: desktop/dashboard.md (hunk 5 above)
Comment (usha-mandya) [reply desktop/dashboard.md]: Updated.

File: desktop/dashboard.md (hunk 1 above)
Comment (mat007) [reply desktop/dashboard.md]: Yes, that looks fine. The only thing I’m wondering is if users will know what’s the VM here, do the docs talk a lot about it in general? Do we not try to hide these kinds of implementation details?

Comment (usha-mandya) [reply desktop/dashboard.md]: Was trying to provide some sort of intro here with a link to the Volumes section for details.

Comment (mat007) [reply desktop/dashboard.md]: Maybe something like «Volumes can be used to store files and share them among containers.»?

Comment (mat007) [reply desktop/dashboard.md]: Or perhaps also mentioning volumes outlive containers?

**Review by mat007 — APPROVED** [review]
Looks good!

**Review by StefanScherer — APPROVED** [review]
LGTM

**Pull request merged**